src/main/java/com/ebivariation/contigalias/dus/NCBIBrowser.java

```diff
@@ -0,0 +1,69 @@
+package com.ebivariation.contigalias.dus;
+
+import org.apache.commons.net.ftp.FTPFile;
+
+import java.io.IOException;
+
+public class NCBIBrowser extends FTPBrowser {
+
+    public static final String NCBI_FTP_SERVER = "ftp.ncbi.nlm.nih.gov";
+    public static final String PATH_GENOMES_ALL = "/genomes/all/";
+
+    public void connect() throws IOException {
+        super.connect(NCBI_FTP_SERVER);
+    }
+
+    public void navigateToSubDirectoryPath(String path) throws IOException {
```

Comment:
As our code style we care about the method order if they are related. We use the more general one first (in this case `navigateToAllGenomesDirectory`) and then the ones that are called from the former (in this case `navigateToSubDirectoryPath`). That makes the code easier to read.

Reply:
`navigateToSubDirectoryPath` is more general as it can navigate to any subdirectory. `navigateToAllGenomesDirectory` is more specialized as it calls `navigateToSubDirectoryPath` to navigate to a certain path. Thus it is below `navigateToSubDirectoryPath`.